backwards-compatability/node8/package.json

```diff
@@ -11,10 +11,6 @@
     "@opentelemetry/sdk-node": "^0.14.0",
     "@opentelemetry/tracing": "^0.14.0"
   },
-  "devDependencies": {
-    "@types/node": "8.10.66",
```

Comment:
I guess this should be kept here otherwise the node 8 test uses node 14 type definitions